wherehows-dao/build.gradle

```diff
@@ -7,4 +7,7 @@ dependencies {
   compile externalDependency.jackson_databind
   compile externalDependency.jackson_core
   compile externalDependency.jackson_annotations
+  compile externalDependency.lombok
```

Comment:
Hibernate needs default no-arg constructor to create entities, also need lombok for other purpose here.